kong_api_authz/README.md

```diff
@@ -99,6 +99,7 @@ form parameter | default | description
 `config.server.port` | _8181_ | The port on wich OPA is listening
 `config.server.connection.timeout` | _60_ | For the connection with the OPA server: the maximal idle timeout (ms)
 `config.server.connection.pool` | _10_ | For the connection with the OPA server: the maximum number of connections in the pool
+`config.document.include_headers` | | Names of request headers to include in the document that is sent to OPA (in the document, all header names will be lower case).
```

Comment:
For those in the know, is this sufficient for understanding how to configure this? I wouldn't know how to specify an array of strings here, but I've also never used this. 🤔

Reply:
Yeah, I reckon so ... if you're using basic Kong API (via `curl` or whatever), then it would just be additional JSON in the configuration:

```.... "document":{"include_headers":["blah","yap","foo"]}} ....```

Even easier if you're using Konga or some other Kong frontend:

![image](https://user-images.githubusercontent.com/23334330/165087941-c04628bb-ddb8-45d7-90d1-9bc23134985a.png)